docs/source/index.rst

```diff
@@ -49,10 +49,20 @@ or in this `presentation <https://indico.cern.ch/event/505613/contributions/2227
    AdministratorGuide/index
    DeveloperGuide/index
    CodeDocumentation/index
+   HowTo/index
+
+
+FAQ & HowTo
+===========
+
+A series of FAQ and HowTo is compiled for various use cases here :doc:`HowTo/index`
 
 Documentation sources
 =====================
 
+
+
+
```

Comment:
The table below has not been updated (the `toctree` abobe is hidden)

Reply:
It would be needed yes, but let's wait to see how the discussion on this section goes.